app/main.js

```diff
@@ -20,3 +20,11 @@ if (process.env.NODE_ENV !== 'production' || process.env.HAS_STYLEGUIDE) {
 }
 
 window.vm = new Vue(vueOptions);
+
+// Set's current theme id to the vuex store read by url param for e.g. ?theme=01
```

Comment:
- [ ] Please add a description for this option to the readme for this component.